homeassistant/components/tahoma/lock.py

```diff
@@ -0,0 +1,91 @@
+"""Support for Tahoma lock."""
+from datetime import timedelta
+import logging
+
+from homeassistant.components.lock import LockDevice
+from homeassistant.const import (
+    ATTR_BATTERY_LEVEL,
+    ATTR_NAME,
+    STATE_LOCKED,
+    STATE_UNLOCKED,
+)
+
+from . import DOMAIN as TAHOMA_DOMAIN, TahomaDevice
+
+_LOGGER = logging.getLogger(__name__)
+
+SCAN_INTERVAL = timedelta(seconds=120)
+
+
+def setup_platform(hass, config, add_entities, discovery_info=None):
+    """Set up the Tahoma lock."""
+    controller = hass.data[TAHOMA_DOMAIN]["controller"]
+    devices = []
+    for device in hass.data[TAHOMA_DOMAIN]["devices"]["lock"]:
+        devices.append(TahomaLock(device, controller))
+    add_entities(devices, True)
+
+
+class TahomaLock(TahomaDevice, LockDevice):
+    """Representation a Tahoma lock."""
+
+    def __init__(self, tahoma_device, controller):
+        """Initialize the device."""
+        super().__init__(tahoma_device, controller)
+        self._lock_status = STATE_UNLOCKED
```

Comment:
Set the lock_status to `None` instead of locked.
```suggestion
        self._lock_status = None
```